app/server.rb

```diff
@@ -453,22 +461,27 @@ def getChildDepts(unit)
   content_type :json
   id = "qt"+shortArk
   item = Item[id]
+  attrs = JSON.parse(Item.filter(:id => id).map(:attrs)[0])
   unitIDs = UnitItem.where(:item_id => id, :is_direct => true).order(:ordering_of_units).select_map(:unit_id)
   unit = unitIDs ? Unit[unitIDs[0]] : nil
 
   if !item.nil?
+    authors = ItemAuthors.filter(:item_id => id).order(:ordering).
+                 map(:attrs).collect{ |h| JSON.parse(h)}
+    citation = getCitation(shortArk, authors, attrs)
     begin
       body = {
         :id => shortArk,
+        :citation => citation,
+        :title => citation[:title],
+        # ToDo: Normalize author attributes across all components (i.e. 'family' vs. 'lname')
+        :authors => authors,
```

Comment:
If we do in fact use the CSL-JSON schema, author needs to use different attribute names than what we have in our DB.  Sticking this here for now so authorlist doesn't break. But in the future it would be best to have just one authorlist sitting in citation.